custom-login/README.md

```diff
@@ -0,0 +1,84 @@
+# Okta React + Custom Login Example
+
+This example shows you how to use the [Okta React Library][] to login a user to a React application.  The login is achieved with the [Okta Sign In Widget][], which gives you more control to customize the login experience within your app.
+
+This example is built with [Create React App][].
+
+## Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Org, you can sign up for one at https://developer.okta.com/signup/.
+* An OIDC application in your Org, configured for Single-Page Application mode. You can find instructions [here][OIDC SPA Setup Instructions].  When following the wizard, use the default properties.  They are are designed to work with our sample applications.
+
+
+## Running This Example
+
+To run this application, you first need to clone this repo and then enter into this directory:
+
+```bash
+git clone git@github.com:okta/samples-js-react.git
+cd samples-js-react/custom-login
+```
+
+Then install dependencies:
+
+```bash
+npm install
+```
+
+You will need to provide these values to the sample application:
+
+* Issuer
+* Client ID
+* Redirect URI
+
+These are application settings for the [OAuth 2.0 Implicit Flow][] and can be found in the Developer Console when looking at the Application that you created earlier.  Place them into this configuration block in  `src/.samples.config.js`:
+
+
+```javascript
+export default {
+  oktaSample: {
+    widget: {
```

Comment:
Any reason we duplicate the config here? Can't we just use the `oidc.issuer` with some split logic?

Reply:
The SIW needs it, and I'd like to not obfuscate that fact.

Reply:
I see your thinking here. I'm thinking more along the lines of keeping this file constant over different iterations of the SIW + framework helper libs. It is a pretty easy fix to parse the `baseUrl` from the `issuer` in the SIW, as noted below.

Personally, the less configuration **I** need to do as a developer - the better.